src/node/convolver.rs

```diff
@@ -155,7 +155,7 @@ impl ConvolverNode {
             channel_config,
         } = options;
 
-        let mut node = context.base().register(move |registration| {
+        let mut node = context.base().base().register(move |registration| {
```

Comment:
Do we need this `.base().base()`, that looks a bit weird

Reply:
That was a runaway vim macro, will fix